.gitignore

```diff
@@ -17,7 +17,7 @@ gulp-tsc-tmp-*
 *.suo
 *.user
 *.sln.docstates
-/.vs
+.vs/
```

Comment:
The new pattern should cover the older pattern too. The leading slash caused the .vs in the root folder to be ignored, the trailing slash causes a .vs folder to be ignored anywhere. The only thing that would not be excluded now that would have been before is a file in root called .vs